project-root/Assets/Scripts/CameraFollow.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace UserInterfaceScripts
+{
+    public class CameraFollow : MonoBehaviour

```

Comment:
This is just a basic camera follow so we can get a feel for how it might look from an isometric camera-style, this is not going to be the camera system that we use obviously.